src/Analyzers/CSharp/Analyzers/UseParameterNullChecking/CSharpUseParameterNullCheckingDiagnosticAnalyzer.cs

```diff
@@ -23,7 +23,7 @@ public CSharpUseParameterNullCheckingDiagnosticAnalyzer()
             : base(IDEDiagnosticIds.UseParameterNullCheckingId,
                    EnforceOnBuildValues.UseParameterNullChecking,
                    CSharpCodeStyleOptions.PreferParameterNullChecking,
-                   CSharpAnalyzersResources.Use_parameter_null_checking,
```

Comment:
Is Use_parameter_null_checking no longer used? I think it should be.

Reply:
This is odd. I'm going to just take some time later and ensure that this analyzer/fixer does something similar as UseNotPattern.

Reply:
`CSharpAnalyzersResources.Use_parameter_null_checking` is still used as the title of the code action. I went through and tried to imitate the `UseNotPattern` fixer and related analyzer as closely as possible. We should be good now.